source/MRViewer/MRUIStyle.cpp

```diff
@@ -173,7 +179,7 @@ bool buttonEx( const char* label, bool active, const Vector2f& size_arg /*= Vect
     const ImRect bb( pos, pos + size );
     ImGui::ItemSize( size, style.FramePadding.y );
     if ( !ImGui::ItemAdd( bb, id ) )
-        return false;
+        return simulateClick;
```

Comment:
i think we should return false here anyway

Reply:
Explained in fleep. ItemAdd returns false for invisible buttons (window is behind other window, or the button is not visible due to scroll, I believe?), which we still want to click.